app/src/main/dist/config/crate.yml

```
@@ -88,8 +88,16 @@ auth.host_based.enabled: true
 #   Note that this authentication method is only available via Postgres
 #   Protocol, meaning that with this configuration, connecting from non-local
 #   hosts via HTTP would not be possible!
+
+# To enable external authentication with JWT globally, configure `iss` and `aud` properties.
+# The CrateDB username is extracted from the `username` field in the JWT token.
+# The `aud` (audience) property is optional and, if not set, defaults to the cluster ID.
+# These settings can be overridden on a per-user level.
 auth:
   host_based:
+#    jwt:
+#      iss: https://example.com
+#      aud: test_aud
```

```suggestion

auth:
  host_based:
    # Node level JWT defaults which are only used if JWT is used as
    # authorization method and users have no more specific value
    jwt:
      # Default URL to a JWK endpoint. E.g. http://example.com;
      iss: null

      # Default audience: The intended recipient the JWT was granted for. Defaults to clusterId
      aud: null
```

The file usually contains the default values that are in use by default.

---

>test_graceful_stop_concurrent_queries (test_decommission.TestGracefulStopDuringQueryExecution) ... WARNING: Using incubator modules: jdk.incubator.vector

>Exception in thread "main" org.elasticsearch.ElasticsearchParseException: null-valued setting found for key [auth.host_based.jwt.iss] found at line number [98], column number [12]

I will comment out